### PR TITLE
OPENEUROPA-1904: Initial commit with module setup.

### PR DESCRIPTION
**Pull request by nagyad**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+workspace:
+  base: /test
+  path: drupal_module_template
```
Comment (imanoleguskiza) [inline .drone.yml]: Usually the path is the name of the component.

File: build-module.sh [hunk 2]
```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
```
Comment (imanoleguskiza) [inline build-module.sh]: We need to remove this file.

File: runner.yml.dist [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+    # Prepare the instance.
+    - "./vendor/bin/drush en toolbar -y"
+    - "./vendor/bin/drush theme:enable bartik -y"
+    - "./vendor/bin/drush theme:enable seven -y"
+    - "./vendor/bin/drush config-set system.theme default bartik -y"
+    - "./vendor/bin/drush config-set system.theme admin seven -y"
+    - "./vendor/bin/drush config-set node.settings use_admin_theme 1 -y"
+    # Enable the modules.
+    - "./vendor/bin/drush en config_devel -y"
+    - "./vendor/bin/drush en drupal_module_template -y"
```
Comment (imanoleguskiza) [inline runner.yml.dist]: Here also, remove or rename :)

File: composer.json [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+    },
+    "repositories": [
+        {
+            "type": "composer",
+            "url": "https://packages.drupal.org/8"
+        }
+    ],
+    "autoload": {
+        "psr-4": {
+            "Drupal\\drupal_module_template\\": "./src/"
```
Comment (imanoleguskiza) [inline composer.json]: Rename to entity_versions.

File: composer.json [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+        }
+    },
+    "autoload-dev": {
+        "psr-4": {
+            "Drupal\\Tests\\drupal_module_template\\": "./tests/"
```
Comment (imanoleguskiza) [inline composer.json]: Rename to entity_versions.

File: runner.yml.dist [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+    settings:
+      file_scan_ignore_directories:
+        - "node_modules"
+        - "bower_components"
+        - "vendor"
+        - "${drupal.root}"
+
+commands:
+  drupal:site-setup:
+    - { task: "symlink", from: "../../..", to: "${drupal.root}/modules/custom/drupal_module_template" }
```
Comment (imanoleguskiza) [inline runner.yml.dist]: Rename to entity_versions.

File: composer.json [hunk 7]
```diff
@@ -0,0 +1,62 @@
...
+        "drupal/console": "^1.6",
+        "drupal/drupal-extension": "~4.0",
+        "drush/drush": "~9.0@stable",
+        "openeuropa/code-review": "~1.0.0-beta2",
+        "openeuropa/drupal-core-require-dev": "^8.7",
+        "openeuropa/task-runner": "~1.0.0-beta5",
+        "phpunit/phpunit": "~6.0"
+    },
+    "scripts": {
+        "post-create-project-cmd": "./build-module.sh",
```
Comment (voidtek) [inline composer.json]: This line needs to be removed.

File: entity_versions.info.yml [hunk 8]
```diff
@@ -0,0 +1,8 @@
+name: Entity versions
+description: Entity versions allows to attach version number to content revisions that help content editors to
+  understand the evolution of the content item. Versions are composed of major, minor and patch numbers that can be
+  changed manually or by third party modules.
+package: OpenEuropa
```
Comment (voidtek) [inline entity_versions.info.yml]: Entity version is a contrib module and this line should be removed.

File: runner.yml.dist [hunk 9]
```diff
@@ -0,0 +1,43 @@
+drupal:
+  root: "build"
+  base_url: "http://web:8080/build"
+  site:
+    name: "OpenEuropa"
+    profile: "minimal"
+  database:
+    host: "mysql"
+    port: "3306"
+    name: "entity_versions"
```
Comment (voidtek) [inline runner.yml.dist]: Is not needed to put this name on the database.
`drupal` is enough.

**Pull request merged**